### ISSUE TITLE
No browser access since installing 4.5.4.0

### ISSUE THREAD

**Issue by henkhend** [MediaBrowser/Emby#3665]
Hi,
I just installed this 4.5.4.0 x64 version on my Asustor NAS. The television app works find, but I cannot connect to Emby using either Firefox or Chromium. The 'trace' in both browser's console is:

Begin ConnectionManager constructor connectionmanager.js:  1:12728
creating ApiClient singleton app.js:  1:7207
Uncaught (in promise) TypeError: connectionManager.getApiClientFromServerInfo is not a function
    createConnectionManager http://juffrouw-janssen:8096/web/app.js?v=4.5.2.0: 1
    promise callback*createConnectionManager/< http://juffrouw-janssen:8096/web/app.js?v=4.5.2.0:1
    promise callback*createConnectionManager http://juffrouw-janssen:8096/web/app.js?v=4.5.2.0:1
    promise callback*start http://juffrouw-janssen:8096/web/app.js?v=4.5.2.0:1
    <anonymous> http://juffrouw-janssen:8096/web/app.js?v=4.5.2.0:1
    <anonymous> http://juffrouw-janssen:8096/web/app.js?v=4.5.2.0:1
app.js:1:7582
Cannot play medium. No decoders for requested formats: audio/mp3 index.html <-- my translation of the Dutch warning.

The Firefox console also reports lots of cookies that will soon be treated as cross-site cookies.

Are these known issues?

Kind regards,
Henk

**Comment by LukePulverenti** [comment]
Hi, can you please supply the complete console output from the browser? Thanks.

**Comment by henkhend** [comment]
Hi Luke,
This is my only Emby install and the TV app started to have issues as well (TV series library not showing), I uninstalled version 4.5.4.0 and reverted to version 4.5.2.0, which works fine, so I cannot reproduce the error right now and I didn't save the console output. I installed the problematic 4.5.4.0 version using the Asustor package on the Emby website. The currently installed and functional 4.5.2.0 version is from the Asustor app store, perhaps that makes a difference?

Regards,
Henk
